docs: avoid child branches on preview branches (#2311)
neondatabase/website: content/docs/guides/vercel.md

4. In the **Vercel integration** drawer, select the **Branches** tab.
5. Check **Automatically delete obsolete Neon branches**.

<Admonition type="note">

> Avoid creating child branches on automatically created preview branches, as it will prevent a preview branch from being automatically deleted. In Neon, child branches must be deleted before you can delete a parent branch. 

Author: ```suggestion
Avoid creating child branches on automatically created preview branches. The presence of a child branch will prevent the parent preview branch from being automatically deleted. In Neon, child branches must be deleted before the parent branch can be deleted.
```

> Avoid creating child branches on automatically created preview branches. This will prevent the preview branch from being automatically deleted. In Neon, child branches must be deleted before you can delete a parent branch. 

Author: ```suggestion
Avoid creating child branches on automatically created preview branches, as it will prevent a preview branch from being automatically deleted. In Neon, child branches must be deleted before you can delete a parent branch. 
```
